unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/WorldChatWindowHUD/PublicChatChannelController.cs

```diff
@@ -108,6 +112,7 @@ public void SendChatMessage(ChatMessage message)
             message.body = $"/w {message.recipient} {message.body}";
 
         chatController.Send(message);
+        socialAnalytics.SendChannelMessageSent(message.sender, message.body.Length, View.channel);
```

Comment:
responsability issues: the controller should solve the `channelId` instead of asking it to the view.. this way you can delete `view.Channel` property

Reply:
You're right! I didn't notice that we already had the `channelId` in the controller. Thanks! Done in https://github.com/decentraland/unity-renderer/pull/2257/commits/eced3e36420d66ba4891f9910b191b650a005f07